src/main/java/org/fundacion/jala/converter/core/ImageConverter.java

```diff
@@ -0,0 +1,129 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ *
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ *
+ * @author Daniela Santa Cruz Andrade
+ */
+package org.fundacion.jala.converter.core;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.fundacion.jala.converter.core.exceptions.ConverterException;
+import org.fundacion.jala.converter.core.parameter.ImageParameter;
+import org.fundacion.jala.converter.core.results.Result;
+import java.io.IOException;
+
+/**
+ * This class converts a image to an specified format.
+ */
+public class ImageConverter {
+    private String startFirstCommand = "ffmpeg -i ";
+    private ImageParameter parameter;
+    private String format;
+    private String output;
+    private String pathOutput;
+    private String outputFileName;
+    private final int HIGH_LIMIT = 1000;
+    private static final Logger LOGGER = LogManager.getLogger();
+    private Result result;
+
+    public ImageConverter(final ImageParameter imageParameter) {
+        this.parameter = imageParameter;
+    }
+
+    /**
+     * Converts the input image.
+     *
+     * @throws ConverterException if process is interrupted.
+     */
+    public void convertImage() throws ConverterException, IOException {
```

Comment:
It should not thrown IOException. Only ConverterException

Reply:
Thank you, I'll fix it